scripts/scil_print_connectivity_filenames.py

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Output the list of filename using the coordinates for a binary connectivity
+matrix. Typically used to move around files that are considered valid after
+the scil_filter_connectivity.py script.
+
+Example:
+scil_print_connectivity_filenames.py filtering_mask.npy labels_list.txt pass.txt
+for file in $(cat pass.txt); do mv ${SOMEWHERE}/${FILE} ${SOMEWHERE_ELSE}/; done
+"""
+
+import argparse
+
+import numpy as np
+
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             load_matrix_in_any_format)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+
+    p.add_argument('in_matrix',
+                   help='Binary matrix in numpy (.npy) format.\n'
+                        'Typically from scil_filter_connectivity.py')
+    p.add_argument('labels_list',
+                   help='List saved by the decomposition script.')
```

Comment:
Just looked at scil_decompose_connectivity.py -h again. Should the --out_labels_list be mandatory??? I didn't even know it could be saved. (just a question)

Reply:
Once generated it is useless to save it everytime and after the end of my many PR this label list (at least for Freesurfer and Brainnetome will be in scilpy/data/)